README.md

```diff
@@ -355,38 +350,35 @@ sudo docker pull mivisionx/ubuntu-16.04
 ``` 
 sudo docker run -it --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-16.04
 ```
-
-  + Optional: Map localhost directory on the docker image
-    - option to map the localhost directory with trained caffe models to be accessed on the docker image.
-    - usage: -v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH} 
-
-``` 
-sudo docker run -it -v /home/:/root/hostDrive/ --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-16.04
-```
-
-**Note:** **Display option with docker**
-
-* Using host display
-
-``` 
-xhost +local:root
-sudo docker run -it --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host --env DISPLAY=unix$DISPLAY --privileged --volume $XAUTH:/root/.Xauthority --volume /tmp/.X11-unix/:/tmp/.X11-unix mivisionx/ubuntu-16.04:latest
-```
-
-* Test display with MIVisionX sample
-
-``` 
-export PATH=$PATH:/opt/rocm/mivisionx/bin
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/mivisionx/lib
-runvx /opt/rocm/mivisionx/samples/gdf/canny.gdf 
-```
+  **Note:** 
+  ##### 1. Map host directory on the docker image
+  
+  * map the localhost directory to be accessed on the docker image.
+  * use `-v` option with docker run command: `-v {LOCAL_HOST_DIRECTORY_PATH}:{DOCKER_DIRECTORY_PATH}`
+  * usage:
+    ``` 
+    sudo docker run -it -v /home/:/root/hostDrive/ --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-16.04
+    ```
+  
+  ##### 2. Display option with docker
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)